app/controllers/preinternship_controller.rb

```diff
@@ -1,7 +1,5 @@
 class PreinternshipController < ApplicationController
-  def home
-  end
+  def home; end
```

Comment:
We use next style for empty methods
```suggestion
  def home
  end
```